templates/serverpod_templates/projectname_server/test/integration/example_endpoint_test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:test/test.dart';
+
+// Import the generated file, it contains everything you need.
+import 'test_tools/serverpod_test_tools.dart';
+
+void main() {
+  withServerpod('Given Example endpoint', (sessionBuilder, endpoints) {
```

Comment:
Discussion: Would it make sense to add a link to our documentation here to read more about out test framework?

Reply:
I don't oppose it, but on the other hand we don't have links in other template files so it would be slightly random perhaps. WDYT?

Reply:
Yeah, you are correct. It might be a new pattern we might not want to introduce.

Reply:
One though here, in the endpoint we actually have a long description that explains how it works. It might be worth adding an explanation here as well explaining what is available and what we are testing.

Reply:
920ebcb